### PR TITLE
add heights element

### PR DESCRIPTION
**Pull request by abhishek97**

### CODE REVIEW

File: sass/styles/_elements.scss [hunk 1]
```diff
@@ -20,4 +20,5 @@
 @import "elements/elements.headings";
 @import "elements/elements.icon";
 @import "elements/elements.fader";
+@import "elements/elements.heights";
```
Comment (aayusharora) [inline sass/styles/_elements.scss]: Why are we having an element for giving 100% height
It should be a class that can be written in any object file where it is required.

Comment (aayusharora) [reply sass/styles/_elements.scss]: @abhishek97

Comment (abhishek97) [reply sass/styles/_elements.scss]: I was not sure where to put it. We will be needing this to fix the wrapping div in ember from affecting the % based calculation of height.

where should such global level classes go? i.e not specific to any object/element.

Comment (aayusharora) [reply sass/styles/_elements.scss]: This is just a css property specific to a object. So, it must be there in the object layer and in scope of a particular object.

There is no object till now which is global object. Which Object needs this class? @abhishek97

**Pull request merged**